src/plugins/usage_collection/common/types/index.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
```

Comment:
Exporting the types as aliased to the unversioned type definitions allows us to use:
`Stats.v1.StatsHTTPBodyTyped` for example